packages/callout/src/Callout.story.tsx

```diff
@@ -81,15 +82,17 @@ export const WithRichContent: ComponentStory<typeof Callout> = ({
   );
 };
 
-export const WithLinks = Template.bind({});
-WithLinks.args = {
-  title: 'Title',
-  children: (
-    <>
-      Lorem Ipsum is simply dummy text &nbsp;
+export const WithLinks: ComponentStory<typeof Callout> = ({
+  // eslint-disable-next-line react/prop-types
```

Comment:
ugh yeah. We should disable this rule project-wide (or at least for story files)